ts/backend/src/app/features/config/defaults.mts

```diff
@@ -21,6 +21,16 @@ export const defaults = {
   redis: {
     url: 'redis://localhost:6379', // redis[s]://[[username][:password]@][host][:port][/db-number]
   },
+  /* MinIO / S3 */
+  minio: {
```

Comment:
S3 instead? We might also have Docker setup with external S3 storage (non-MinIO)?